roles/zuul/files/jobs/hoist.yaml

```diff
@@ -10,7 +10,7 @@
           #!/bin/bash -ex
           source /opt/venvs/ansible/bin/activate
           ansible-playbook -i inventory/nodepool.py install-ci.yml --skip-tags monitoring -e @secrets.yml.example
-          ansible-playbook -i inventory/nodepool.py tests/validate-ci.yml
+          ansible-playbook -i inventory/nodepool.py tests/files/validate-ci.yml
```

Comment:
@gandelman-a: Thanks for the pointer! I'll drop this file (and any other jobs that are no longer needed) in another pull request.